ee-9/feature-pack/pom.xml

```diff
@@ -863,6 +863,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.spec.jakarta.xml.soap</groupId>
+            <artifactId>jboss-saaj-api_2.0_spec</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
I've just found this must be replacing the existing, but this dependency is still declared at ee-9/feature-pack/pom.xml

```
<dependency>
            <groupId>org.jboss.spec.javax.xml.soap</groupId>
            <artifactId>jboss-saaj-api_1.4_spec</artifactId>
            <scope>provided</scope>
</dependency>
```

Reply:
That's the only issue I've found.

We don't have a Jira validating the uses of the JBoss forks instead of the eclipse ones for this spec, see https://issues.redhat.com/browse/WFLY-15098, so I assume that is intentional.

Reply:
Oops. Removed this dependency.   Fork this saaj api is required , because we need backport this jboss module support change : https://github.com/jboss/jboss-jakarta-saaj-api_spec/commit/21665f66c8eda4b621f30b84da0a99a32b955548

Reply:
ahh, cool, thanks for confirming it, then we should be fine with this.